apps/felvin-faq/src/index.ts

```diff
@@ -0,0 +1,14 @@
+import { InstantApp } from "@felvin-search/core";
+import { Component, queryToData } from "./App";
+
+const App: InstantApp = {
+  id: "@felvin-community/felvin-faq",
+  name: "Felvin FAQ",
+  description: "Shows answers to common Felvin FAQ questions",
+  queryToData,
+  Component,
+  // screenshotPath: "./files/screenshot.png",
+  // exampleSearchQueries: [],
```

Comment:
Let's add these, would have to add all the questions as example search queries.

Reply:
Thanks for pointing this out.